src/block/columns/style.js

```diff
@@ -39,10 +119,10 @@ const BlockStyles = memo( props => {
 
 	return (
 		<>
+			<Styles { ...props } />
 			<Alignment.Style { ...props } { ...alignmentOptions } />
 			<BlockDiv.Style { ...props } />
 			<MarginBottom.Style { ...props } />
-			<ContainerDiv.Style { ...props } { ...containerDivOptions } />
```

Comment:
I feel that we're using some styling from the Container Div (maybe in the layout tab, something with the size/max width/spacing). Unless you are sure that we do not use this, and if it interferes with the new styles, then I think you can retain this.